src/tests/stress/pedsim/Ped_util/Global_funs.enc

```diff
@@ -0,0 +1,315 @@
+-- This file was automatically converted by encorec
+
+module Global_funs
+import Agent_passive
+import XML
+fun invSqrt(input : int) : real
+  var ret = 0.0
+  EMBED (unit)
+    {
+  float x = #{input};
+  float xhalf = 0.5f * x;
+  int i = *(int*)&x;
+  i = 0x5f3759df - (i >> 1);
+  x = *(float*)&i;
+  #{ret}= x*(1.5f - xhalf*x*x);
+  }
+  END
+  ret
+end
+fun find_extreme(agents : [Agent]) : (int, int, int, int)
+  var extreme = (-(10000), -(10000), 10000, 10000)
```

Comment:
I'll just re-write it to not contain magic numbers, it's just a lazy way to initialize the extreme value tuple.